Cargo.toml

```diff
@@ -10,11 +10,11 @@ repository = "https://github.com/MagicalBitcoin/rust-hwi"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-bitcoin = { version = "0.25", features = ["use-serde", "base64"] }
-serde = { version = "^1.0", features = ["derive"] }
-serde_json = { version = "^1.0" }
-strum_macros = "0.18.0"
-base64 = "^0.11"
+bitcoin = { version = "0.28", features = ["use-serde", "base64"] }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = { version = "1.0" }
```

Comment:
You removed the `^`, it should still be there

Reply:
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements.

I did notice that, but was being lazy so I left it like that xD. Fixed now.